src/main/java/cs361/battleships/models/Board.java

```diff
@@ -10,6 +10,7 @@
 
 	@JsonProperty private List<Ship> ships;
 	@JsonProperty private List<Result> attacks;
+	@JsonProperty private Weapon spaceLaser = new Weapon();
```

Comment:
Nice, adding the weapon here allows for easier implementation of weapons in the future. Great job!